pandas/io/excel/_base.py

```diff
@@ -755,6 +759,28 @@ class ExcelWriter(metaclass=abc.ABCMeta):
     >>> with ExcelWriter("path_to_file.xlsx", mode="a", engine="openpyxl") as writer:
     ...     df.to_excel(writer, sheet_name="Sheet3")
 
+    Here, the `if_sheet_exists` parameter can be set to replace a sheet if it
+    already exists:
```

Comment:
does this work with `.to_excel` directly?

Reply:
IOW are the docs correct there?

Reply:
What is IOW?

Reply:
But no, it is exposed to the ExcelWriter only and then that class checks for `self.if_sheet_exists` inside the `to_excel` method.